ament_package/url.py

```diff
@@ -16,9 +16,9 @@
 class Url(object):
     __slots__ = ['url', 'type']
 
-    def __init__(self, url, type_=None):
+    def __init__(self, url, *, type=None):
```

Comment:
Yeah, `type_` as a keyword argument is not great. While `type` is also not a good choice I think it is slightly better. Or does anybody have a better proposal for a name which fits well but is not already a type?